distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -134,6 +134,7 @@ ALERT;Twitter Binding has become X. All related references to Twitter and Tweets
 ALERT;AndroidTV Binding: The thing configuration of 'port' has been renamed and split to 'googletvPort' and 'shieldtvPort'
 ALERT;evcc Binding: Update to evcc API version 0.123.1 results in several new or updated channels. Existing Items will need to be adjusted.
 ALERT;ISM8 Binding: Most channels have changed and are now using Units of Measurements. Items must be adapted and the things must be recreated.
+ALERT;JavaScript Scripting Automation: The 'Item' suffix has been removed from Item.type, i.e. 'SwitchItem' becomes 'Switch'.
```

Comment:
Can you also document the action users need to do e.g. update rules? Is this optional or mandatory?

Reply:
That’s quite difficult because there is much you can do with this.
E.g. for logging, this change doesn’t matter, but when using the equals operator to check for a given Item type, you would need to update one side of the comparison.